.github/.linkspector.yml

```diff
@@ -0,0 +1,15 @@
+dirs:
+  - docs
+ignorePatterns:
+  - pattern: '://localhost'
+  - pattern: '://.*.?example\\.com'
+  - pattern: 'developer.github.com'
+  - pattern: 'docs.github.com'
+  - pattern: 'github.com/<your_github_handle>'
```

Comment:
Let's check for GitHub links, too, to detect if some upstream docs have moved or a repo has been deleted/renamed, etc.
```suggestion
```

Reply:
Yeah, I just copied these from the old config, these sound questionable too:
```
  - pattern: 'tailscale.com'
  - pattern: 'wireguard.com'
  ```

Reply:
Yes. Let us remove all, and we can add if we need to.